lib/insights.ts

```diff
@@ -66,6 +68,10 @@ class AlgoliaAnalytics {
   _userHasOptedOut: boolean;
   _cookieDuration: number;
 
+  // user agent
+  _ua: string = "";
```

Comment:
`_ua`  is the current name in the search client too
https://github.com/algolia/algoliasearch-client-javascript/blob/ead86654beb7bc5215bb8ff66d4c951637190a10/src/AlgoliaSearchCore.js#L175